<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DLGShapeAdderTri</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Coordinate 2</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Coordinate 3</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Y</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="5" column="3">
    <widget class="QLineEdit" name="lineEdit_4"/>
   </item>
   <item row="10" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Coodinate 1</string>
     </property>
    </widget>
   </item>
   <item row="24" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="22" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="24" column="3">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>ADD TRIANGLE</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="4" column="3">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="8" column="2">
    <widget class="QLineEdit" name="lineEdit_5"/>
   </item>
   <item row="8" column="3">
    <widget class="QLineEdit" name="lineEdit_6"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
